src/models/historical_location.js

```diff
@@ -32,7 +32,8 @@ module.exports = (sequelize, DataTypes) => {
     time: { type: DataTypes.DATE, allowNull: false }
   });
 
-  // XXX relations
+  // Relations with Location and Thing are already specified on
```

Comment:
Sequelize adds an `id` filed automagically so we get the `findById` for free. In addition to that, adding the relations on `Locations` and `Things` we get a `Locations.getHistoricalLocations` and `Things.getHistoricalLocations`. It may be worth to add also `HistoricalLocations.getLocations` and `HistoricalLocations.getThing`